spec/requests/api/chargebacks_spec.rb

```diff
@@ -71,4 +71,36 @@ def app
     )
     expect_request_success
   end
+
+  it "can create a new chargeback rate detail" do
+    api_basic_authorize
+    run_post rates_url, :rate => 0, :enabled => true
+
+    actual = @result["results"].first
+    expect(actual["rate"]).to eq("0")
+    expect(actual["enabled"]).to be true
+    expect_request_success
+  end
+
+  it "can edit a chargeback rate detail" do
+    chargeback_rate_detail = FactoryGirl.create(:chargeback_rate_detail, :rate => 0)
+
+    api_basic_authorize
+    run_post rates_url(chargeback_rate_detail.id), gen_request(:edit, :rate => 0.02)
+
+    expect(@result["rate"]).to eq("0.02")
+    expect_request_success
+    expect(chargeback_rate_detail.reload.rate).to eq("0.02")
+  end
+
+  it "can delete a chargeback rate detail" do
+    chargeback_rate_detail = FactoryGirl.create(:chargeback_rate_detail)
+
+    api_basic_authorize
+
+    expect {
+      run_delete rates_url(chargeback_rate_detail.id)
+    }.to change(ChargebackRateDetail, :count).by(-1)
+    expect(last_response.status).to eq(204)
+  end
```

Comment:
once the role identifiers are added, the api_basic_authorize will need to be called with the appropriate action_identifier(:rates, :action_name).